sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Release History
 
-## 7.3.0-beta.2 (Unreleased)
+## 7.2.1 (2021-05-12)
+### Fixed
```

Comment:
nit: `Fixes`

Reply:
I am following this guidelines for the  changelog
https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#change-logs